src/PrestaShopBundle/Controller/Admin/Sell/Catalog/ManufacturerController.php

```diff
@@ -141,9 +142,53 @@ public function bulkDeleteAction(Request $request)
         return $this->redirectToRoute('admin_manufacturers_index');
     }
 
-    public function bulkEnableManufacturerAction()
+    /**
+     * Enables manufacturers on bulk action
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function bulkEnableAction(Request $request)
```

Comment:
@sarjon, @matks - Any ideas to decrease code duplication in bulkEnable/Disable actions?

Reply:
Code duplicate is OK here because it is different controller actions 👍 it makes the actions very readable